ReadMe.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/AtamaZack/ah-frontend-dojo.svg?branch=task%2F%23165273520-travis-ci-setup)](https://travis-ci.com/AtamaZack/ah-frontend-dojo) [![Maintainability](https://api.codeclimate.com/v1/badges/bb975aa777e8d8d8909a/maintainability)](https://codeclimate.com/github/AtamaZack/ah-frontend-dojo/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/bb975aa777e8d8d8909a/test_coverage)](https://codeclimate.com/github/AtamaZack/ah-frontend-dojo/test_coverage)
```

Comment:
@AtamaZack, I would like to find out whether you were able to add our repository to code climate. These badges should reference the repository on the Andela organization. I know for now the badges will not be showing any results until the PR is merged but it would be better to point them to our repository.